src/main/webapp/i18n/de/team.json

```diff
@@ -61,7 +61,11 @@
                 "noResults": "Keine Studenten im Kurs gefunden für Suche:"
             },
             "errors": {
-                "studentAlreadyAssignedToTeam": "Student mit Login {{ studentLogin }} ist bereits in Team mit Id {{ teamId }."
+                "studentAlreadyAssignedToTeam": "Student mit Login {{ studentLogin }} ist bereits in Team mit Id {{ teamId }.",
```

Comment:
While we're here: `{{ teamId }` does not look correct, can we fix that if it is an issue?

Reply:
I couldn't find where it is used. There is studentsAlreadyAssignedToTeams that is coming from server but not studentAlreadyAssignedToTeam. It is not as trivial as adding an s so I will not touch this. We can open an issue for this, because this is not related to importing and this pull request is already pretty big.